src/utils/files.ts

```diff
@@ -0,0 +1,2 @@
+export const getFileExt = (fileName: string): string | undefined =>
```

Comment:
i think default exports make sense when the file is descriptive of the contents (eg: `ForbiddenError.ts` exports `class ForbiddenError`). In this case, because this is a utility file, it's likely to get more exported functions in the future so idt i want to change it. wdyt?

Reply:
sure, am ok with this rationale